content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -1,67 +1,236 @@
 ===============================
 
-Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams` and enable the after sales
-options: *Refunds, Returns, Coupons, Repairs and Onsite Interventions*.
+Go to :menuselection:`Helpdesk --> Configuration --> Teams` to select a :guilabel:`Team`. Scroll to
+the :guilabel:`After-Sales` section and choose which of the following options to enable:
 
-.. image:: after_sales/aftersales1.png
+- :guilabel:`Refunds`: Issues credit notes to refund a customer, or adjust the remaining amount due.
+- :guilabel:`Coupons`: Offers discounts and free products through an existing coupon program.
+- :guilabel:`Returns`: Initiates a product return from a customer through a reverse transfer.
+- :guilabel:`Repairs`: Creates repair orders for broken or faulty products.
+- :guilabel:`Field Service`: Plans onsite intervention through the Field Service application.
+
+.. image:: after_sales/after-sales-enable.png
+   :align: center
+   :alt: View of a team's settings page with a focus on the after-sales services.
+
+.. warning::
+   As all of the After-Sales services require integration with other applications, enabling any of
+   them may result in the installation of additional modules or applications. *Installing a new
+   application on a One-App-Free database will trigger a 15-day trial. At the end of the trial, if a
+   paid subscription has not been added to the database, it will no longer be accessible.*
+
+Issue a refund with a credit note
+=================================
+
+A *credit note* is a document issued to a customer informing them that they have been credited a
+certain amount of money. They can be used to provide a full refund to a customer, or to adjust any
+remaining amount due. While they are usually created through the :menuselection:`Accounting` or
+:menuselection:`Invoicing` applications, they can be created through a :menuselection:`Helpdesk`
+:guilabel:`Ticket`.
+
+.. note::
+   Invoices must be posted before a credit note can be generated.
+
+To create a credit note from a :menuselection:`Helpdesk` :guilabel:`Ticket`, click on the
+:guilabel:`Refund` button in the upper left corner. Select the corresponding invoice from the
+:guilabel:`Invoices to Refund` drop-down.
+
+.. image:: after_sales/after-sales-refund-details.png
+   :align: center
+   :alt: View of a refund creation page.
+
+Choose a :guilabel:`Credit Method` from one of the following options:
+
+- :guilabel:`Partial Refund`: The credit note is created in draft and can be edited before being
+  issued.
+- :guilabel:`Full Refund`: The credit note is auto-validated and reconciled with the invoice. *This
+  is the option to choose if a validated invoice needs to be canceled*.
+- :guilabel:`Full refund and new draft invoice`: The credit note is auto-validated and reconciled
+  with the invoice. The original invoice is duplicated as a new draft. *This is the option to choose
+  if a validated invoice needs to be modified.*
```

Comment:
nice use of italics to help the reader understand their options 👍 
same formatting as earlier comment: lowercase after colon and no periods on list items